docs/03-computation/03-how-it-works.mdx

```diff
@@ -0,0 +1,44 @@
+---
+title: How It Works
+sidebar_position: 2
+tags: ['explainer']
+---
+
+Based on the configuration you provided, the Privacy Computation Engine creates a set of **Processing Records** that contain:
+
+- Identification of the data field in your database. Table/Object and field/property. E.g. `USER`, `e_mail_address`
+- A [Privacy Scope](https://github.com/blindnet-io/product-management/blob/main/refs/schemas/priv/RFC-PRIV.md#privacy-scope) e.g. (`CONTACT.EMAIL` x {`COLLECTION`, `STORING`, `USING`} x {`SERVICES`,`COMPLIANCE`})
+- One or more [Legal Bases](https://github.com/blindnet-io/product-management/blob/main/refs/schemas/priv/RFC-PRIV.md#legal-base) e.g. a legal base of type `CONTRACT`
+
+It is possible to end-up having serval records for one Data Category, e.g.
+
+<!-- prettier-ignore -->
+| Data in database | Privacy Scope | Legal bases |
+| :----: | :-----: | :--------|
+| table:`USER` field:`name` | `NAME.FIRST-NAME` x {`COLLECTION`, `STORING`, `USING`} x {`SERVICES`,`COMPLIANCE`} | `CONTRACT` |
+| table:`USER` field:`e_mail_address` | `CONTACT.EMAIL` x {`COLLECTION`, `STORING`, `USING`} x {`SERVICES`,`COMPLIANCE`} | `CONTRACT` |
+| table:`USER` field:`e_mail_address` | `CONTACT.EMAIL` x `USING` x {`MARKETING`,`ADVERTISING`} | `LEGITIMATE-INTEREST`|
+| table:`USER` field:`e_mail_address` | `CONTACT.EMAIL` x `SHARING` x {`ADVERTISING`} | `CONSENT`|
+
+The records given in the example correspond to the situation of a System that is processing the user's e-mail address for purposes of providing services to the user and for compliance with regulation (KYC) under `CONTRACT` legal base.
+The same System is sending marketing and advertising e-mail to the user under `LEGITIMATE-INTEREST` assuming that as long as the user don't not object they might be interested in additional services.
+The System only shares the e-mail with partners under `CONSENT` legal base.
+
+A **Transfer Record** has:
```

Comment:
Delete this section and add it later, when we have transfers.

Reply:
This is only a copy past of what Milan previously wrote.

I'll follow your advice and comment this section for now.